common/app/views/support/HtmlCleaner.scala

```diff
@@ -298,8 +298,17 @@ case class LiveBlogLinkedData(isLiveBlog: Boolean)(implicit val request: Request
         el.select(".block-time.published-time time").foreach { time =>
           time.attr("itemprop", "datePublished")
         }
-        el.select(".block-elements").foreach { body =>
-          body.attr("itemprop", "articleBody")
+        el.select(".block-title").foreach { title =>
+          title.attr("itemprop", "headline")
+
+        }
+        el.select(".block-elements").foreach { blockElements =>
```

Comment:
good although my only worry is this has to be done before the author cleaner, maybe not worth adding a comment about it but I do worry it's not 100% obvious from the code?  or maybe it worth commenting it?